src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -175,6 +179,7 @@ export class SkyAgGridService {
         columnMoveRight: this.getIconTemplate('arrows'),
         columnMovePin: this.getIconTemplate('arrows')
       },
+      onCellKeyPress: this.onKeyPress,
```

Comment:
Assigning an instance method directly to an object like this can lead to hard-to-debug issues later if `this` were to sneak into the body of that function. I'd either move the contents of `onKeyPress` into a fat-arrow function here or use a fat-arrow function to call through to `onKeyPress` (`onCellKeyPress: (keypress) => this.onKeyPress(keypress)`).